### ISSUE TITLE
Code Table Request - new life stage: spore

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#6855]
### Goal
_Describe what you're trying to accomplish. This is the only necessary step to start this process. The Committee is available to assist with all other steps. Please clearly indicate any uncertainty or desired guidance if you proceed beyond this step._

Record life stage (currently age class in Arctos) using GBIF terms for an incoming collection to keep their published data consistent.

### Context
_Describe why this new value is necessary and existing values are not._

No existing term properly describes this life stage which is a term in the GBIF vocabulary.

### Table
_Code Tables are http://arctos.database.museum/info/ctDocumentation.cfm. Link to the specific table or value. This may involve multiple tables and will control datatype for Attributes. OtherID requests require BaseURL (and example) or explanation. Please ask for assistance if unsure._

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctage_class

### Proposed Value
_Proposed new value. This should be clear and compatible with similar values in the relevant table and across Arctos._

spore

### Proposed Definition
_Clear, complete, non-collection-type-specific **functional** definition of the value. Avoid discipline-specific terminology if possible, include parenthetically if unavoidable._

A spore is a reproductive structure that is adapted for dispersal and surviving for extended periods of time in unfavorable conditions. Spores form part of the life cycles of many bacteria, plants, algae, fungi and some protozoans. A chief difference between spores and seeds as dispersal units is that spores have very little stored food resources compared with seeds. Spores are usually haploid and unicellular and are produced by meiosis in the sporangium by the sporophyte. Once conditions are favorable, the spore can develop into a new organism using mitotic division, producing a multicellular gametophyte, which eventually goes on to produce gametes. Many ferns, especially those adapted to dry conditions, produce diploid spores. In this case spores are the units of asexual reproduction, because a single spore develops into a new organism. By contrast, gametes are the units of sexual reproduction, as two gametes need to fuse to create a new organism. |  http://rs.gbif.org/vocabulary/gbif/life_stage/spore

### Collection type
_Some code tables contain collection-type-specific values. ``collection_cde`` may be found from https://arctos.database.museum/home.cfm_

Herb, Teach

#### Attribute data type
_If the request is for an attribute, what values will be allowed?
free-text, categorical, or number+units depending upon the attribute (TBA)_

N/A

#### Attribute controlled values
_If the values are categorical (to be controlled by a code table), add a link to the appropriate code table. If a new table or set of values is needed, please elaborate._

N/A

#### Attribute units
_if numerical values should be accompanied by units, provide a link to the appropriate units table._

N/A

#### Part preservation attribute affect on "tissueness"
_if a new part preservation is requested, please add the affect it would have on "tissueness": No Influence, Allows, or Denies_

N/A

### Priority
_Please describe the urgency and/or choose a priority-label to the right. You should expect a response within two working days, and may utilize [Arctos Contacts](https://arctosdb.org/contacts/) if you feel response is lacking._

Part of OGL data migration

### Example Data
_Requests with clarifying sample data are generally much easier to understand and prioritize. Please attach or link to any representative data, in any form or format, which might help clarify the request._

### Available for Public View
_Most data are by default publicly available. Describe any necessary access restrictions._

yes

### Helpful Actions

- [x] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [x] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators @happiah-madson @falco-rk 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
This seems fine. Is it a value that will be used more than a few times in their data?

**Comment by Jegelewicz** [comment]
I just summarized the data and it is only used in three records, but I have trouble finding anything in the code table that could be used in its place. Ideas appreciated!

**Comment by dustymc** [comment]
As a life stage, are spore (or some cases thereof??) and seed different? The definition seems to be somewhere between BIO101 and a part definition while avoiding life stage....

>, or  anything in the code table that could be used in its place 

It'd sure be nice to have some 'if the number be less than...then thou shalt....' guidance but maybe three makes this a good candidate for https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#age

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Part search spore returns a few things - I think we actually added a part name spore when migrating, a few other collections have spore in part remark. This might be a good opportunity to get everyone to record this the same way.

**Comment by Jegelewicz** [comment]
> This might be a good opportunity to get everyone to record this the same way.

Agree! And honestly, spore does seem more appropriate as a part name, but as a life stage what is it? zygote?

> Spores form part of the life cycles of many plants, algae, fungi, and some protozoans. They are typically unicellular and haploid, being produced by meiosis. Being units of asexual reproduction, spores do not combine with other spores, but directly form new haploid organisms.

**Comment by Jegelewicz** [comment]
Yes - these are hard to define, but they are clearly a life stage - any ideas?

**Comment by dustymc** [comment]
>any ideas?

[Not really](https://github.com/ArctosDB/arctos/issues/6856#issuecomment-1781291045) - but denormalizing generally finds a way to bite back.

>hard to define

Just at full-scale. I HATE the "when bla then this, when blaugh then that, when...." homonym nonsense (it unavoidably means the data can't do the one thing controlled vocabularies DO, find all of the stuff of interest and nothing else), but it's probably that or 'bacterial spore' and 'sexual spore' and .... and I guess, having typed it out, I like that better than the alternative. (But this still feels like it might be lost and wandering part-stuff.)

**Comment by Jegelewicz** [comment]
> denormalizing

It's not my fault that the scientific community has named two things with the same term! That is where the denormalization comes from....

This is why the GBIF definition is so long.

**Comment by dustymc** [comment]
> why the GBIF definition is so long.

GBIF is dealing with an exchange standard and has to accommodate whatever comes in. We are dealing with basically the opposite of that. Certainly following some bigger standard-ish-thing makes sense when possible, but we are NOT doing the same things and we do NOT have similar data models or constraints. (Nor are we doing the same thing as other less Community-oriented management systems, from which "spore" might mean a single thing.)

**Comment by Jegelewicz** [comment]
> accommodate whatever comes in

So do we? And then some because we also have art, cultural objects, and geology.

**Comment by dustymc** [comment]
>So do we?

[No!](https://github.com/ArctosDB/arctos/issues/6855#issuecomment-1781326726) Errr, we do, but not all in the same lump (and not necessarily all in this lump - where exactly does https://arctos.database.museum/info/ctDocumentation.cfm?table=ctage_class bleed off into https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#age?). And it's a question - is "spore" meaning 73 things, or 73 things that mean 'spore,' less-evil? I think maybe the latter right now....

**Comment by Jegelewicz** [comment]
> where exactly does https://arctos.database.museum/info/ctDocumentation.cfm?table=ctage_class bleed off into https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#age?). 

Where exactly does either of those bleed into https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_part_name?

Without a definition for the tables, we are left to make haphazard decisions.

**Comment by dustymc** [comment]
>definition for the tables

AHA (maybe)! I think one of your Issues (that I can't find...) just made sense. I'm still not convinced that *tables* (incl. references) don't define themselves - https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type is what it is - but https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#age / https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#age_class should (and don't) answer my question; definitions should be functional.

**Comment by Jegelewicz** [comment]
>  https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type is what it is

Is it though? I think the line between that and https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type is pretty darn fuzzy

**Comment by dustymc** [comment]
>pretty darn fuzzy

Not really - one clearly and structurally applies to the concept (the individual, or WHATEVER got cataloged) and one clearly and structurally applies to 'things to which one can stick barcodes.' Using them incorrectly for whatever reason doesn't change that. We should clearly be saying "this liver tested positive for cooties" and "this individual has cooties" as two (potentially independent) THINGS, and maybe when we all have infinite resources we will....

**Comment by happiah-madson** [comment]
Okay, here I think we're also okay with just using spore as the part name and choosing not to enter 'life stage' data. Is that ok? As long as it's somewhere, I think we're happy.

**Comment by Jegelewicz** [comment]
noted and closing!
